### ISSUE TITLE
DOCS | Trening og optimaliseringsprosess | Treningsstrategi

### ISSUE THREAD

**Issue by martinstereo** [kartverket/BirdAI#114]
### Oppgavebeskrivelse
**Arbeidsoppgave:** skrive om hvordan disse tingene her: 

**Sprint 4**
- [ ] Treningsoppsett (hardware, programvare, rammeverk)
- [ ] Augmenteringsteknikker som ble brukt for å øke datasettet og forbedre modellens robusthet
- [ ] Treningsregime (antall epoker, early stopping kriterier, osv.)

Senere sprint:
- Valg av treningsalgoritme og begrunnelse for dette valget
- Hyperparametere som ble brukt (learning rate, batch size, optimizer, osv.) og hvorfor disse ble valgt
- Tap-funksjoner som ble brukt (f.eks. focal loss, IoU loss) og begrunnelse
- Valideringsmetoder (k-fold cross-validation, hold-out validation)
- Teknikker for å forhindre overfitting (regularisering, dropout, osv.)
- Eventuelle transfer learning strategier (om du brukte pre-trenede modeller)

Påvirket resultatene.

**Kontekst:**

### Definition of Done (DoD)
**Krav for fullført oppgave:** Når en har forklart og begrunnet hvordan disse kriteriene påvirket resultatet for modellen vår.
